src/Unleash.php

```diff
@@ -14,4 +14,8 @@ public function isEnabled(string $featureName, ?Context $context = null, bool $d
     public function getVariant(string $featureName, ?Context $context = null, ?Variant $fallbackVariant = null): Variant;
 
     public function register(): bool;
+
+    public function getFeatures(): array;
+
+    public function getFeature(string $featureName);
```

Comment:
We cannot add these two methods to the interface for now, it would be a breaking change (meaning anyone who was already implementing the interface would suddenly get fatal errors if they updated the package).

What should be done is adding these as soft requirement - meaning only as a docblock comment as for example here: https://github.com/Unleash/unleash-client-php/blob/main/src/Configuration/Context.php#L10

That way the methods are not enforced but people have time to add them in their implementations. When next major version is released, these soft-enforced methods will be moved to the real methods.